core/spec/helpers/base_helper_spec.rb

```diff
@@ -71,6 +71,76 @@
     end
   end
 
+  describe '#spree_storefront_resource_url' do
+    let!(:store) { create(:store) }
+    let!(:taxon) { create(:taxon) }
+    let!(:product) { create(:product) }
+
+    context 'for Product URL' do
+      context 'when storefront is not used' do
+        before do
+          allow(helper).to receive(:frontend_available?).and_return(false)
```

Comment:
@MatthewKennedy we can skip this line and change the description as we always going to have storefront, but not necessary the rails one :)